### PR TITLE
Feature/updcmeps

### PR DESCRIPTION
**Pull request by DeniseWorthen**
- Updates to latest ESCOMP/master

- Fixes minor bug in FB_diagnose 

- Removes return code from LOGMSG and associated chkerr calls

- minor code cleanup

No changes to baselines. Logs are posted at associated UFS-S2S PR [163](https://github.com/ufs-community/ufs-s2s-model/pull/163)

### CODE REVIEW

**Comment by DeniseWorthen** [comment]
One wcoss regression test failed due to 'ice_gather_scatter' error at startup. This is a random error which I have only seen after switching to the rt.sh system. I don't think I ever saw it when using the CompsetRunner. 

I am re-running all the wcoss tests now and will post new logs.

The same test failed again on wcoss. I then ran the develop branch and it also failed in the same test on wcoss. Issue [#164](https://github.com/ufs-community/ufs-s2s-model/issues/164)

File: mediator/med_fraction_mod.F90 [hunk 1]
```diff
@@ -682,7 +682,7 @@ subroutine med_fraction_set(gcomp, rc)
 
        call FB_getFldPtr(is_local%wrap%FBImp(compice,compice) , 'Si_ifrac', Si_ifrac, rc=rc)
        if (ChkErr(rc,__LINE__,u_FILE_u)) return
-       call FB_getFldPtr(is_local%wrap%FBImp(compice,compice) , 'Si_imask', Si_imask, rc=rc)
+       call FB_getFldPtr(is_local%wrap%FBImp(compice,compice) , 'Si_imask' , Si_imask, rc=rc)
```
Comment (junwang-noaa) [inline mediator/med_fraction_mod.F90:685]: remove space?

File: mediator/med_fraction_mod.F90 [hunk 2]
```diff
@@ -721,60 +721,39 @@ subroutine med_fraction_set(gcomp, rc)
...
+       if (is_local%wrap%comp_present(compatm)) then
 
+          if (trim(coupling_mode) == 'nems_orig' ) then
+             maptype = mapnstod_consf
           else
-
              if (med_map_RH_is_created(is_local%wrap%RH(compice,compatm,:),mapfcopy, rc=rc)) then
-                ! TODO (mvertens, 2019-11-20): this is not being used when ice and atm are on the same grid
-                ! - this needs to be implemented
                 maptype = mapfcopy
```
Comment (junwang-noaa) [inline mediator/med_fraction_mod.F90:731]: I am curious, do we need this?

File: mediator/med_methods_mod.F90 [hunk 3]
```diff
@@ -163,7 +163,7 @@ subroutine med_methods_FB_RWFields(mode,fname,FB,flag,rc)
         enddo
 
         call med_methods_FB_diagnose(FB, 'read '//trim(fname), rc)
-        if (present(flag)) flag = .true.
+	if (present(flag)) flag = .true.
```
Comment (junwang-noaa) [inline mediator/med_methods_mod.F90:166]: should we keep the space?

File: mediator/med_fraction_mod.F90 (hunk 2 above)
Comment (DeniseWorthen) [reply mediator/med_fraction_mod.F90:731]: I suspect this has to do with one of the CESM non-active configurations,  when atm and ice are the same grid. I don't think this is something we will need.

File: mediator/med_fraction_mod.F90 (hunk 1 above)
Comment (DeniseWorthen) [reply mediator/med_fraction_mod.F90:685]: I will double check the several white space changes. They should all be in the merge to ESCOMP and will go into our emc/develop w/ this PR.

**Review by binli2337 — APPROVED** [review]

**Review by junwang-noaa — APPROVED** [review]

**Comment by DeniseWorthen** [comment]
I just want to confirm: We are OK to merge this to emc/develop even though the single test failed on dell-p3?

**Comment by junwang-noaa** [comment]
I think we need to fix the issue before we can commit. If develop branch
has the same issue, we need to fix that as an emergency commit before we do
any future commit.

**Comment by DeniseWorthen** [comment]
All regression tests now pass. Logs are posted at S2S PR [#163](https://github.com/ufs-community/ufs-s2s-model/pull/163)

**Pull request merged**